pyproj/_transformer.pyx

```diff
@@ -202,6 +203,11 @@ cdef class _TransformerGroup:
                 operation_factory_context,
                 allow_ballpark,
             )
+            proj_operation_factory_context_set_discard_superseded(
+                self.context,
+                operation_factory_context,
+                int(not bool(allow_superseded)),
```

Comment:
Does this work?
```suggestion
                not allow_superseded,
```

Reply:
Yep, it looks like it works. I wasn't sure whether Cython had the same handling of truthy values as Python.